contrib/ci/dependencies.xml

```diff
@@ -5,6 +5,19 @@
       <package />
     </distro>
   </dependency>
+  <dependency type="build" id="secureboot-db">
+    <distro id="ubuntu">
+      <package />
+    </distro>
+    <distro id="debian">
+      <package />
+    </distro>
+  </dependency>
+  <dependency type="build" id="dbxtool">
+    <distro id="fedora">
+      <package />
+    </distro>
+  </dependency>
```

Comment:
I think we should talk about what to do for the snap too here, should we rely upon the host distro to provide the database or ship it with the snap (my inclination is to ship it with the snap; we can keep control of it then which is nice)

Reply:
I did wonder if fwupd should keep a copy too, but I'm not particularly enamoured with the idea of shipping some static security database in RHEL and being on the hook to update the package at a moments notice for CVEs. Maybe we ship a fallback copy if the other sources are not found?

Reply:
> Maybe we ship a fallback copy

This is what I did.